Sources/MessagingInApp/Gist/Network/GistQueueNetwork.swift

```diff
@@ -22,10 +22,14 @@ class GistQueueNetworkImpl: GistQueueNetwork {
             throw GistNetworkRequestError.invalidBaseURL
         }
 
+        let sdkClient = DIGraphShared.shared.sdkClient
+
         var urlRequest = URLRequest(url: baseURL.appendingPathComponent(request.path))
         urlRequest.httpMethod = request.method.rawValue
         urlRequest.addValue(state.siteId, forHTTPHeaderField: HTTPHeader.siteId.rawValue)
         urlRequest.addValue(state.dataCenter, forHTTPHeaderField: HTTPHeader.cioDataCenter.rawValue)
+        urlRequest.addValue(sdkClient.sdkVersion, forHTTPHeaderField: HTTPHeader.cioClientVersion.rawValue)
+        urlRequest.addValue(sdkClient.source.lowercased(), forHTTPHeaderField: HTTPHeader.cioClientPlatform.rawValue)
```

Comment:
Same question as for Android. Did you verify if React Native Data Pipelines support is sending expected source and version correctly? If yes, then we're good with the change.